slateratom/lib/globals.f90

```diff
@@ -152,6 +152,9 @@ module globals
   !> exc energy density on grid
   real(dp), allocatable :: exc(:)
 
+  !> average local, effective potential
```

Comment:
```suggestion
  !> average local effective potential equivalent to non-local GSK potential (if present)
```

Reply:
Done in revised version (with GSK <--> GKS I assumed).